common-test/src/main/java/org/alfresco/hxi_connector/common/test/docker/repository/AlfrescoRepositoryContainer.java

```diff
@@ -122,6 +124,7 @@ protected void configure()
                         -Dmetadata-keystore.aliases=metadata
                         -Dmetadata-keystore.metadata.password=oKIWzVdEdA
                         -Dmetadata-keystore.metadata.algorithm=DESede
-                        """.replace("\n", " "));
+                        """.replace("\n", " "))
+                .withLogConsumer(new Slf4jLogConsumer(LoggerFactory.getLogger(AlfrescoRepositoryContainer.class.getSimpleName())));
```

Comment:
This log consumer here is redundant - you can remove it

Reply:
done!